hometasks/4-functional-programming/src/components/Sort/dataSorter.ts

```diff
@@ -0,0 +1,16 @@
+import { SortType } from 'src/components';
+import { Row } from '../../components/Table/Table';
+
+export const sortData = (data: Row[], sortType?: SortType): Row[] => {
+    if(!sortType) return data;
+    let sortedData: Row[];
+    switch(sortType) {
+        case SortType.asc: 
+            sortedData = [...data].sort((a, b) => a.lastPayments - b.lastPayments);
+            break;
+        case SortType.desc:
+            sortedData = [...data].sort((a, b) => b.lastPayments - a.lastPayments);
```

Comment:
NIT: Similar to filtering, you can do object with handlers mapping
```
cons SORTERS = {
  asc: (a, b) => a.lastPayments - b.lastPayments,
  desc: (a, b) => b.lastPayments - a.lastPayments
}
```

Then you won't need this ambiguous switch construction:

```
export const sortData = (data: Row[], sortType?: SortType): Row[] => {
   const sorter = SORTER[sortType];
   return [...data].sort(sorter);
};
```